src/structure/options.html.in

```diff
@@ -16,6 +16,7 @@
                         <input class="form-check-input" id="space-filling" type="checkbox" />
                         <label class="form-check-label" for="space-filling" title="show atoms as spheres">space filling</label>
                     </div>
+		   </div>
```

Comment:
are you sure about this one? Where we missing a closing tag?

Reply:
Nice catch!